hack/components/bump-bridge-marker.sh

```diff
@@ -92,6 +93,7 @@ echo 'Get bridge-marker image name and update it under CNAO'
 BRIDGE_MARKER_TAG=$(git-utils::get_component_tag ${BRIDGE_MARKER_PATH})
 BRIDGE_MARKER_IMAGE=quay.io/kubevirt/bridge-marker
 BRIDGE_MARKER_IMAGE_TAGGED=${BRIDGE_MARKER_IMAGE}:${BRIDGE_MARKER_TAG}
-sed -i "s#\"${BRIDGE_MARKER_IMAGE}:.*\"#\"${BRIDGE_MARKER_IMAGE_TAGGED}\"#" \
+BRIDGE_MARKER_IMAGE_DIGEST="$(docker-utils::get_image_digest "${BRIDGE_MARKER_IMAGE_TAGGED}" "${BRIDGE_MARKER_IMAGE}")"
```

Comment:
`""` wrapping is not needed (already wrapped in `docker-utils::get_image_diges`). please remove it:

```
BRIDGE_MARKER_IMAGE_DIGEST=$(docker-utils::get_image_digest "${BRIDGE_MARKER_IMAGE_TAGGED}" "${BRIDGE_MARKER_IMAGE}")
sed -i -r "s#\"${BRIDGE_MARKER_IMAGE}(@sha256)?:.*\"#\"${BRIDGE_MARKER_IMAGE_DIGEST}\"#" \
```

Reply:
Why? the linter prefer it like that, and I don't understand what wrong with it. It causing no harm.

Reply:
I misread `docker-utils::get_image_digest`. please ignore this comment.